src/main/scala/firrtl/passes/RemoveCHIRRTL.scala

```diff
@@ -10,6 +10,8 @@ import firrtl.Utils._
 import firrtl.Mappers._
 import firrtl.options.Dependency
 
+import scala.collection.mutable
+
```

Comment:
why mutable here? originally it wasn't mutable.

Reply:
It is my understanding that it kind of was: https://underscore.io/blog/posts/2020/05/14/porting-and-seq.html

Reply:
fixed in https://github.com/freechipsproject/firrtl/pull/1796/commits/4265ed79eff1e5106a1dfab25b30466c17839ae3